src/pages/components/lead-space-block.mdx

```diff
@@ -22,29 +24,76 @@ import ResourceLinks from 'components/ResourceLinks';
 
 ## Overview
 
-When designed and implemented effectively, the Lead space block can be used to engage visitors immediately in one of the
-most important moments in the user’s journey through a complex web environment.
+When designed and implemented effectively, lead space block can be used to engage visitors immediately upon page load. Lead space block is one of the most streamlined beginnings to a page.
 
-It includes a headline, subheadline, supporting message,
-[Link list](https://www.ibm.com/standards/carbon/web-components/?path=/story/components-linklist--default), and a call to action.
-There is also the option to add media in the form of an image or video.
+It includes a heading, subheading, supporting media,
+[link list](https://www.ibm.com/standards/carbon/web-components/?path=/story/components-linklist--default), and a call to action. Note that most elements in the lead space block are optional, meaning they can be hidden or enabled as needed.
 
-![Default](../../images/pattern/lead-space-block/leadspace-block.jpg)
+<Row>
 
-<br />
+<Column colMd={8} colLg={8}>
+
+![Lead space block with heading, subheading, video, link list, button, and bottom border](../../images/component/lead-space-block/lead-space-block-overview.png)
+
+</Column>
+
+</Row>
+
+1. **Heading:** Heading has the highest information hierarchy in a lead space block, and should be succinct and descriptive.
+2. **Subheading:** Optional copy can be added for additional description.
+3. **Media:** An optional image or video that provide more context to the lead space block.
+4. **Link list:** Optional link list to related resources.
+5. **CTA:** Optional CTA to lead the user to additional resources.
+6. **Bottom border:** Optional border to help separate the lead space block from the following [content section](../components/content-section) or [content block](../components/content-block).
+
+## Modifiers
+
+### Heading highlight
+
+Like other lead spaces, lead space block's heading can be modified to highlight a word or phrase. This can be used to highlight product names, industries, or particular value. The highlighted word or phrase can be anywhere in the heading, but it should only ever appear once in a single lead space. This only applies to the heading, and does not extend to any other lead space block element.
```

Comment:
```suggestion
Like other lead spaces, lead space block's heading can be modified to highlight a word or phrase. This can be used to highlight product names, industries, or a particular value. The highlighted word or phrase can be anywhere in the heading, but it should only ever appear once in a single lead space. This only applies to the heading, and does not extend to any other lead space block element.
```